resources/code/cfn/lf-emr-workshop.template

```diff
@@ -558,7 +569,15 @@
                   "Ref":"EC2KeyPair"
                }
             },
-            "Name":"LF-EMRCluster",
+            "Name": {
```

Comment:
If your intention is to have multiple EMR clusters for different IdP providers in the same account  then changing the cluster name alone is not sufficient in this CF as three other roles ( LakeFormationSAMLRole,LakeFormationEMRRole,LakeFormationEMRInstanceProfileRole) it depends are not going to be unique.

Reply:
I know, it's not meant for that. It just adds a little bit clarity through the cluster name, in case the audience wants to try out one after one, they can just look at the cluster name to find out what IdP was used for that cluster.